versions.tf

```diff
@@ -18,6 +18,6 @@ terraform {
   required_version = ">= 0.12.6"
 
   required_providers {
-    google = "~> 3.15"
+    google = "~> 3.30.0"
```

Comment:
Are we depending on some `3.30` feature? I maybe wrong but I think `~> 3.30.0` will lock the module to `>=3.30.0, <3.31.0`